views/pages/organizations.ejs

```diff
@@ -10,4 +10,5 @@
     </div>
         <p style="text-align: center;" class="black-text">Organizations Page - wouldn't have this picture, but wanted to show different options.</p>
 </body>
+<%- include('../partials/footer')-%>
 </html>
```

Comment:
@ZM202 Lets remove the closing html tag and add them into the footer.ejs file.

Reply:
All set.